packages/shared/src/components/icons/index.ts

```diff
@@ -1,3 +1,5 @@
+// When adding a new icon, please add it alphabetically to the list below.
```

Comment:
This never works 😆

Reply:
maybe this hell? https://www.npmjs.com/package/eslint-plugin-sort-exports and enable it only for this file

Reply:
You're right, replaced with eslint rule instead